Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -370,9 +372,10 @@ - (void)initSessionWithLaunchOptions:(NSDictionary *)options isReferrable:(BOOL)
         if (![options.allKeys containsObject:UIApplicationLaunchOptionsURLKey] &&
             ![options.allKeys containsObject:UIApplicationLaunchOptionsUserActivityDictionaryKey]) {
             
+            self.asyncRequestCount = 0;
+            BOOL willNotCheckAsyncAttributionAPIs = ![self checkFacebookAppLinks] && ![self checkAppleSearchAdsAttribution];
```

Comment:
Do you mean the short circuit `&&` here?  If a Facebook deferred applink is being checked, then it will _*not*_ check for Apple Search Ads.

Maybe this is clearer and doesn't short-circuit:
```objc
self.asyncRequestCount = 0;
BOOL checkingFacebook = [self checkFacebookAppLinks];
BOOL checkingAppleAds = [self checkAppleSearchAdsAttribution];

if (!(checkingFacebook || checkingAppleAds)) {
...
```